packages/components/organisms/f-content-cards/README.md

```diff
@@ -122,108 +122,108 @@ The following rudimentary example can be used as a guide for implementing this c
+cards source(s)
+
+### `error`
+
+The `error` slot can be used by the consuming code to surface a message to the user when an error has been
+encountered upon initialising the cards source instance
+
+
 ## Configuration
 
 The following props can be used to configure the component:
 
 ### `api-key`
 
 **Type:** string
+
 **Required:** false
 
 The Braze SDK api key.
 
 > If no apiKey is provided the component will look for an existing appboy implementation at `window.appboy`.
 
-### `brands`
-
-**Type:** array
-**Required:** false
-
-List of brands for user's current postcode.
-
-> Only required when displaying Home Promotion Card 1 and 2
-
 ### `user-id`
 
 **Type:** string
+
 **Required:** false
 
 The Braze User ID associated to the current authenticated user.
 
 > If no userId is provided the component will look for an existing appboy implementation at `window.appboy`.
 
-### `enabled-card-types`
+### `custom-cards`
 
 **Type:** array
-**Required:** false
-
-An array of custom content card types to display.
-
-> If no array is passed the component will default to showing all supported content card types.
-
-### `locale`
 
-**Type:** string
-**Required:** true _if_ [vue-i18n](https://kazupon.github.io/vue-i18n/) plugin not used by consuming application
+**Required:** false
 
-Locale in `lang_COUNTRY` format - e.g. `en-GB`
+Can be used to inject custom card data in the event that no data is received from braze.
 
 ### `pushToDataLayer`
 
 **Type:** function
+
 **Required:** false
 
 A callback for feeding back analytics regarding content cards to the consuming application
 
 > If no function is passed then this will be replaced with a noop function
 
-### `testId`
+### `locale`
 
 **Type:** string
-**Required:** false
-
-Indicates the test id attribute of the component root element.
-
-> If this is missing or nully, all child components will also be rendered without test id
-> attributes.
 
-### `showLoadingState`
+**Required:** true _if_ [vue-i18n](https://kazupon.github.io/vue-i18n/) plugin not used by consuming application
 
-**Type:** boolean
-**Required:** false
+Locale in `lang_COUNTRY` format - e.g. `en-GB`
 
-Whether or not to show a skeleton loading state whilst initialising Braze. Defaults to `true`.
-For configurations where the enabled card types includes cards other than Post Order cards, the
-number of loading state cards shown is limited to the lowest of either `3` or the limit set in
-`cardLimit`
+### `testId`
 
-### `cardLimit`
+**Type:** string
 
-**Type:** number
 **Required:** false
 
-Limit the amount of cards to show. Defaults to `-1` which shows all cards.
+Indicates the test id attribute of the component root element.
+
+> If this is missing or nully, all child components will also be rendered without test id
+> attributes.
 
 ## Events
 
 The following events can be emitted by the component, with the shape given:
 
-### `@voucherCodeClick`
+### `@voucher-code-click`
```

Comment:
While we're here is it possible to covert this to a table. Like [this](https://github.com/justeat/fozzie-components/tree/master/packages/components/molecules/f-searchbox#available-events)